plugins/modules/notification/discord.py

```diff
@@ -154,13 +149,12 @@ def main():
 
     if info['status'] != 204:
         result['http_code'] = info['status']
-        result['stderr'] = info['msg']
+        result['msg'] = info['msg']
         result['failed'] = True
```

Comment:
No need to add this, ansible will automagically add a `failed` output value when you call `fail_json()`